OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,201 @@
+## Building Oracle JDK (Server JRE) base image
+
+Please refer README.md under docker/OracleJava for details on how to build Oracle Database image.
+
+https://github.com/oracle/docker-images/tree/master/OracleJava
+
+
+## Building Oracle FMW Infrastructure Docker Install Image
+
+Please refer README.md under docker/OracleFMWInfrastructure for details on how to build Oracle FMW Infrastructure image.
+
+https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure
+
+
+
+## Building Docker Image for SOA
+
+IMPORTANT: To build the SOA image, you must first download the Oracle SOA Suite 12.2.1.3 binary and drop in folder `../OracleSOASuite/dockerfiles/12.2.1.3`. The binaries can be downloaded from the Oracle Software Delivery Cloud (https://edelivery.oracle.com). Search for "Oracle SOA Suite" and "Oracle Service Bus", download fmw_12.2.1.3.0_soa.jar and fmw_12.2.1.3.0_osb.jar and drop it under `dockerfiles/12.2.1.3`. 
+
+* [Oracle Software Delivery Cloud](https://edelivery.oracle.com)
+
+$ sh buildDockerImage.sh -v 12.2.1.3
+
+   Usage: buildDockerImage.sh -v [version]
+   Builds a Docker Image for Oracle SOA Suite.
+
+
+Verify you now have the image `oracle/soa:12.2.1.3` in place with 
+
+$ docker images
+
+
+IMPORTANT: the resulting images will NOT have a domain pre-configured. But, it has the scripts to create and configure a soa domain while creating a container out of the image.
+
+
+## Creating a container for AdminServer
+
+Start a container to launch the Admin Server from the image created using above steps. The environment variables used to configure the domain are defined in adminserver.env.list file. Replace in adminserver.env.list the values for the Database and WebLogic passwords. 
+
+Create an environment file adminserver.env.list
+
+    CONNECTION_STRING=<Database Container Name>:<port#>/<ORACLE_PDB>
+    RCUPREFIX=<RCU_Prefix>
+    DB_PASSWORD=<database_sys_password>
+    DB_SCHEMA_PASSWORD=<soa-infra schema password>
+    ADMIN_PASSWORD=<admin_password>
+    MANAGED_SERVER=<managed servername>
+    DOMAIN_TYPE=<soa/osb/bpm>
+    
+Sample Data will look like this:
+
+    CONNECTION_STRING=soadb:1521/soapdb
+    RCUPREFIX=SOA1
+    DB_PASSWORD=Welcome1
+    DB_SCHEMA_PASSWORD=Welcome1
+    ADMIN_PASSWORD=Welcome1
+    MANAGED_SERVER=soa_server1
+    DOMAIN_TYPE=soa
+    
+To start a docker container with a SOA domain and the WebLogic AdminServer call `docker run command` and pass the above `adminserver.env.list` file.
+
+A sample docker run command is given below:
+
+     $docker run -i -t  --name soaas --network=SOANet -p 7001:7001  -v /scratch/DockerVolume/SOAVolume/SOA:/u01/oracle/user_projects   --env-file ./adminserver.env.list oracle/soa:12.2.1.3
+     
+The options "-i -t" in the above command runs the container in interactive mode and you will be able to see the commands running in the container. This includes the command for RCU creation, domain creation and configuration followed by starting the Admin Server. 
```

Comment:
Please use back ticks around `"-i -t"`

This seems like the appropriate place to highlight that the image does not have a domain created and that it's done on the first run of the AdminServer container.